client/plots/gsea.js

```diff
@@ -185,6 +185,7 @@ class gsea {
 	async main() {
 		this.config = structuredClone(this.state.config)
 		this.settings = this.config.settings.gsea
+		this.imageUrl = null // Reset the image URL
```

Comment:
Should create a key to hold all instance props that will be cleared in `main()`, to make it easier to track/remember, and may be able to automate this later as part of `rx` dispatch cycle.

```js
   this.clearedProps = {.}
  
 main() {
    ...
    for(const k of Object.keys(this.clearedProps)) delete this.clearedProps[k]
    ..
 }
 
 // in other functions, can set arbitrary props under this.clearedProps
 ```

Reply:
This is not my plot. I just added the minimum fixes I needed to solve the jira ticket. As these are improvements I suggest we talk about them when you are back and we include Robin in the conversation? I am not sure I follow you completely. Aslo I dont want to take more time from you and I would rather minimize changes prone to errors as I am not familiar with the GSEA logic.